specifyweb/frontend/js_src/lib/components/FormEditor/Editor.tsx

```diff
@@ -73,6 +74,8 @@ export function FormEditor(): JSX.Element {
       ? userText.switchToHorizontalLayout()
       : userText.switchToVerticalLayout();
 
+  const unloadProtects = React.useContext(UnloadProtectsContext)!;
```

Comment:
why not just move this line into the UseLabelsSchema component, rather than do prop drilling?

Reply:
good question... I should have had